sdk/resourcemanager/internal/testutil/helper_test.go

```diff
@@ -29,11 +30,52 @@ func TestCreateDeleteResourceGroup(t *testing.T) {
 	ctx := context.Background()
 	cred, options := GetCredAndClientOptions(t)
 	subscriptionID := GetEnv("AZURE_SUBSCRIPTION_ID", "00000000-0000-0000-0000-000000000000")
-	StartRecording(t, pathToPackage)
+	defer StartRecording(t, pathToPackage)()
```

Comment:
I personally prefer this which is more explicit, but your choice
```suggestion
	stop := StartRecording(t, pathToPackage)
	defer stop()
```